package.json

```diff
@@ -16,7 +16,7 @@
     "jasmine": "^2.3.0",
     "phantomjs": "^1.9.11",
     "sinon": "^1.10.3",
-    "uglify-js": "2.3.6",
+    "uglify-js": "2.6.3",
```

Comment:
@bjacobel or @andy-armstrong - are there any large concerns with this upgrade, or additional testing I can do? This is used in the Makefile for `make javascript`, I already ran that and the minified js had no differences at all.